Utilities/DB.php

```diff
@@ -3,5 +3,124 @@
 
 class DB
 {
+    private $serverdb = 'mysql:host=192.168.20.20; dbname=cat-test';
+    private $username = 'root';
+    private $password = '';
+
+    /**
+     * Create DB connection
+     * @return PDO
+     */
+    public function dbConnect()
+    {
+        $dbconnect = new PDO($this->serverdb, $this->username, $this->password);
+        $dbconnect->setAttribute(
+            PDO::ATTR_DEFAULT_FETCH_MODE,
+            PDO::FETCH_ASSOC);
+        return $dbconnect;
+    }
+
+    /**
+     * @return array
+     */
+    public function getCatBreed()
+    {
+        $curl = curl_init();
+        curl_setopt_array($curl, array(
+            CURLOPT_URL => "https://api.thecatapi.com/v1/breeds/",
+            CURLOPT_CUSTOMREQUEST => "GET",
+            CURLOPT_RETURNTRANSFER => true,
+            CURLOPT_HTTPHEADER => array(
+                'x-api-key: ec254e44-3996-458b-8522-4933954d8fcd'
+            ),
+        ));
+
+        $response = curl_exec($curl);
+        $error = curl_error($curl);
+        curl_close($curl);
+
+        $responseArray = json_decode($response, true);
+        $breedArray = [];
+        $breedNameArray = [];
+        for ($i = 0; $i < count($responseArray); $i++) {
+            array_push($breedArray, $responseArray[$i]["id"]);
+            array_push($breedNameArray, $responseArray[$i]["name"]);
+        }
+        $associativeBreedArray = array_combine($breedArray, $breedNameArray);
```

Comment:
This array combining thing can be cleaned up - try something like `$associativeBreedArray[$responseArray[$i]["id"]] = [$responseArray[$i]["name"];` inside the for loop.

Reply:
clean up should be done in s2t3 branch I believe?

Reply:
Yes, if you like